zerver/management/commands/mobile_push_service.py

```diff
@@ -153,3 +160,17 @@ def _handle_register_subcommand(self, *args: Any, **options: Any) -> None:
             print("- Return to the documentation to learn how to test push notifications")
         else:
             print("Mobile Push Notification Service registration successfully updated!")
+
+    def _handle_deactivate_subcommand(self, *args: Any, **options: Any) -> None:
+        request = {
+            "zulip_org_id": settings.ZULIP_ORG_ID,
+            "zulip_org_key": settings.ZULIP_ORG_KEY,
+        }
+        self._log_params(request)
+
+        response = self._request_push_notification_bouncer_url(
+            "/api/v1/remotes/server/deactivate", request
+        )
+
+        if response.json()["result"] == "success":
+            print("Mobile Push Notification Service registration successfully deactivated!")
```

Comment:
We should display some sort of error if this doesn't succeed, right?

Reply:
(Ideally, just the error response returned by the server).

Reply:
Or if `_request_push_notification_bouncer_url` ensures it can't have failed, we should have an assert rather than an `if` statement.